volumegroup/README.md

```diff
@@ -47,7 +47,7 @@ If a volume group corresponding to the specified volume group name already exist
 parameters in the CreateVolumeGroupRequest, the Plugin MUST reply 0 OK with the corresponding CreateVolumeGroupResponse.
 CSI Plugins MAY create the following types of volume groups:
 
-Create a new empty volume group. Note that N volumes with some backend label Y could be considered to be in "group Y"
+Create a new empty volume group or a group with specific volumes. Note that N volumes with some backend label Y could be considered to be in "group Y"
 which might not be a physical group on the storage backend. In this case, an empty group can still be created by the CO
 to hold volumes. After the empty group is created, create a new volume, specifying the group name in the volume.
```

Comment:
This isn't an option, maybe remove it?

> create a new volume, specifying the group name in the volume

Or explain that `ModifyVolumeGroupMembership` needs to be called to add the volume afterwards.